### DISCUSSION TITLE
暑假規劃，到底要幹嘛

### DISCUSSION THREAD

**Discussion by quan0715** [1101-GS4719-C/Python-Community-GS4719-C#1509] [General]
暑假快到了，每次跟同學討論，他們就會說暑假要精進自己，瘋狂做side  project，讀英文考托福，暑假要到 google/IBM/台積電實習等等，就好羨慕大家，好像只有我暑假想瘋狂睡覺之類的，怎麼辦，這裡開放報團取暖拜託拜託

（更）暑假想要做一些 side project跟python 還有notion api 操作有關的，不知道大家有沒有興趣，如果有興趣的話歡迎加個好友來討論一下。
> 想個笑話，才能讓機器人給我高分

<img width="376" alt="image" src="https://github.com/NCU-GS4719-Python/Python-Community/assets/60366187/6dd2ee62-c286-4e82-b440-f2a5cb6e55b6">

**Comment by poweichen00** [comment]
我也蠻想嘗試寫Notion api ，我平常就有使用Notion去記錄東西，感覺蠻有趣的。

**Comment by Tzuhsuannnnn** [comment]
我暑假有參加學校的一個暑期交換，想去國外看看，感覺蠻酷的。BTW我之前也一直很焦慮大家暑假好像都有很厲害的事要做，只有我每天都在玩，但我媽有一天跟我說，未來就要工作那麼久了，幹嘛不好好珍惜一下剩下的幾個暑假，我覺得蠻有道理的所以推薦你如果想幹嘛就直接去做吧。

**Comment by Chiuhsianglin** [comment]
我會去打工，打工可以增加很多見聞，而且我認為暑假有很長的時間，感覺可以好好規劃要做什麼，不然每次耍廢到最後都會發現這個假期好像什麼都沒做又要開始上課了。

**Comment by PaulSu0905** [comment]
我想利用暑假的時間多出去玩，但也想花時間精進自己，目前有想到特別要做的事是準備英檢考試，還在努力思考找其他自己有興趣的事去做，因為暑期實習沒上所以時間很多QQ，

**Comment by Evening218** [comment]
每次暑假我都覺得有好多事情想做,但是最後都會變成一種無限循環,跟自己說好不容易學期結束了,給自己放一周的假應該很合理,然後就變成每天吃吃睡睡耍廢暑假就過完了QAQ

**Comment by MaxChi870** [comment]
在考慮打工，不過大部分時間應該會是開始趕補習進度、耍廢跟做專題，另外有在計畫準備語言相關的檢定考，像多益

**Comment by Miinteaa** [comment]
我應該會一邊打工一邊學開車、或者說閱讀課外讀物增進自己，因為學期間能夠分給這些的時間真的太少了，終於有一個可以供我好好利用的大段空閒，當然要好好規劃！另外還想要和朋友一起出遊攝影！希望我們能約到一個好時間，拍到很多好看的照片

**Comment by zhiron544** [comment]
我有報名AWS的課程跟八月要開始做大學專題，另外就是吃吃喝喝?

**Comment by 110302001** [comment]
我這個暑假有規劃出國交換，趁有長一點的假可以好好體驗國外的生活跟練習語言，我們教授都跟我們說這是我們最後一個暑假要好好珍惜，所以如果還有空我應該會去學開車、或把上學期的課再複習一次，因為這學期的課太重，有些都學的不太好，還好蠻多老師上課都有錄影片，決定暑假再看一次。

**Comment by ThanatosJun** [comment]
剛剛樓上也有提到「未來就要工作那麼久了，幹嘛不好好珍惜一下剩下的幾個暑假」這件事，我也曾經想過這個問題，不如說這正是我在煩惱的事。現在我的面前有兩條路，一條是去實習，順利地通過前兩周的話下學期平日也要上班，一條是在學校上課的同時也學習自己喜歡的課外技能，例如 Unity 和 AI 運用，找幾個同學做個小遊戲，並用 html 等搭建自己的CV。我認為每次暑假都是人的一次大蛻變，所以選擇才如此困難！

**Comment by jim1743** [comment]
我的暑假規劃可能是嘗試去找看看打工，因為打工可以加深外面的見聞，提早知道出社會的感覺，另外也會嘗試學習有關於專題的研究，畢竟暑假的時間很長，若沒有好好地充實自己，開學容易被身旁的同儕給超越。

**Comment by tzurou** [comment]
我這個暑假的目標是把專題計畫的實證做完，然後寫推甄的備審資料，準備開學後的口試，可以的話我也想去報駕訓班學開車，培養自己的技能，上大學後發現好多人都會開車，時間允許的話再去考雅思，因為之後想要出國交換，但會花比較多時間和錢，所以還很猶豫，暑假要做的事情很多，但又想留一點時間出去玩，也想應屆就推上，好煎熬

**Comment by YOGA012** [comment]
我的暑假規劃大概就是平時打工存錢，再利用下班後的時間去駕訓班練車，考到汽車駕照後跟朋友們出去玩個幾天，然後再和和家人出國玩玩，順便到國外的大學走走逛逛增廣見聞，平時大家都各忙各的，難得有空就一起聚一聚放鬆一下，大概就是這樣。

**Comment by titan92601** [comment]
暑假到了，大多數同學會做side project、考托福、實習，難免會感到壓力。我認為每個人都有不同的需求，適當休息也是一種進步。你想做跟Python和Notion API相關的side project很棒。

**Comment by 20050201Shawn** [comment]
我目前沒啥專題可做，可能只會準備多益跟打工，剩下時間都在耍廢，但想想這學期沒什麼空閒時間，就覺得暑假耍廢蠻合理的。

**Comment by SusuTima** [comment]
暑假要進實驗室用專題，然後順便學開車，最重要的是要把學期中間欠的睡眠全部補回來，我的肝縮水到快要不見了*_*。

**Comment by alexender0104** [comment]
暑假被教授留下來做專題，目前的規劃可能是早上去駕訓班練車，下午呆實驗室做專題，晚上回家還要繼續讀多益，這個暑假感覺跟沒放一樣啊！

**Comment by Shang621** [comment]
暑假是一個寶貴的休閒時間，可以安排各種有意義的活動。你可以計劃一些旅行，探索新的地方，體驗不同的文化和風景。或者，你可以選擇參加一些夏令營或志願者活動，擴展自己的技能和社交圈。另外，暑假也是自我成長和放鬆的時候，可以閱讀一些好書，學習新技能，或者尋找一些興趣愛好。重要的是，結合自己的興趣和目標，找到適合自己的方式度過一個充實而有意義的暑假。

**Comment by poiu6fi** [comment]
每次暑假前都想好自己要幹嘛幹嘛，結果一到暑假的時候就整個沒動力，直接躺在家裡耍廢

**Comment by lilishh** [comment]
暑假我會喜歡回家看看我在外讀書的時候家鄉有什麼改變。然後在家浪費時間找一些平常沒時間看得劇或電影看完，或是跟朋友約出去玩，總之有很多事情可以做啦。

**Comment by yayun601506** [comment]
我要先跑系上的營隊，再去考駕照、考多益，然後放假就是要出去玩呀!!
通常寒暑假我都會找短期打工，多賺錢才有錢出去玩~

**Comment by cassy0201** [comment]
我這個暑假安排了參加WAT也是人生第一次離開台灣那麼長的時間到那麼遠的地方，又期待又緊張阿啊啊啊

**Comment by HirokiMantissa** [comment]
暑假就是留在學校做畢業專題，點點點，教授沒有要讓我過暑假。

**Comment by 710388** [comment]
把握機會充實自己，順便賺點錢，還有思考一下未來的道路怎麼走

**Comment by tshsu04224** [comment]
我這個暑假要去公司暑期實習，想藉著這個時間增加一下自己的見聞與能力，體驗企業真實的工作環境，以往的暑假我也都宅在家追劇、耍廢，感覺可以趁此機會做一些有意義的事。

**Comment by YeeDaniel** [comment]
蠻厲害的，不過我覺得出門玩也是一種有意義的事情，某方面來說，也算是犒勞自己辛苦了一個學期啦~對我來說，暑假其實也是一個蠻好充實自己的機會，不論是學開車、考英檢、學習知識，都是很好的規劃

**Comment by OHoln** [comment]
我認為暑假是一個良好習慣的機會，我打算在暑假建立健康的作息及運動習慣。也希望能在暑假預習未來會用到的開發框架Electron

**Comment by buoioiry** [comment]
暑假確實是很好用來學習的時候，但我沒有那麼有上進心，只想好好放假，可能會去探索一些未來職涯吧，但不想給自己太多事情做，出社會之後就不一定有暑假了，現在不出去玩未來要再有那麼長的假期感覺不太容易

**Comment by MingWei174** [comment]
可以去學校附近打工學習一些社會經驗，或者參加一些營隊培養團隊精神並且學會獨立，如果還有閒暇時間可以上一些有學分的網課，大三大四可以過得輕鬆些!

**Comment by hunteryang0108** [comment]
暑假是一個很好的學習新技能的時間，學期間因為白天要上課，所以很難參加到一些學術的研討會或課程，而暑假期間白天時間可以自己安排利用，所以一定要好好把握機會。我在今年利用暑假報名了張家凱老師的暑期先修課程，學習Python以及生成式AI，我覺得這樣的暑假安排非常充實。

**Comment by Nana1218** [comment]
到暑假前，我總是會給自己很遠大的目標，有很多想做的事情，然而實際過完後，會發現好像真正能完成的事情很少，但我也發現，如果是報名一些課程類的東西，會比較能幫助我完成目標。

**Comment by kingsophy** [comment]
我在暑假最常和班上同學一起去公園打球，因為平常課業繁忙，可以打球的時間只有一個禮拜一次的體育課，真的很少，因此每到暑假就是我打球的快樂時光。

**Comment by hunteryang0108** [comment]
暑假時我參加了多門的課程增進自己的課外知識，現在我正在參與中央大學張家凱老師的生成式人工智慧課程，我覺得在這堂課上收穫了許多東西，非常推薦各位同學積極參與。

**Comment by Jerryguo888** [comment]
在這個暑假，我會想上大學先修，大學先修課程很有挑戰性，但也是個很好的機會讓我提早接觸大學的內容，利用這些先修課，我不僅可以測試自己的學習能力，還能在進入大學前打下更堅實的基礎。

**Comment by william-60760** [comment]
我自己每一年的暑假規劃則是會依據學習階段改變。高一高二通常是比較休閒的安排，如出遊、看小說、玩遊戲等等。高三和大一則是會往精進自己的方向安排，如參與先修課程。基本上我覺得休閒和忙碌的暑假交替安排挺適合自己的。

**Comment by ShuliANN** [comment]
回顧這個暑假，覺得自己應該更懂得運用時間，在沒有其他課業影響的時候，去培養興趣，或學一個新的技能，比如說：練吉他，學剪輯影片等等。另外，我覺得其實出國玩也很不錯，趁現在還有長長的假期，可以去體驗不同生活，探索不同的文化！
雖然看到大家都在實習，很焦慮......＠＠

**Comment by lai1008** [comment]
我應該會去以前的補習班打工，同時可以跟我的學弟妹分享一些之前讀書的經驗，或是給他們一些讀書上的建議，然後還可以從中幫助他們解一些題目。

**Comment by 113607514** [comment]
去年暑假，我選擇去打工，這不僅讓我賺了一些錢，也增加了不少社會經驗，還能接觸到不同的人和事，這也是一種成長。同時，我也試著給自己留出休息的時間，讓自己放空、做一些平時忙碌時無法做的事，像是看書、和朋友聚會，或者就單純地放鬆。

**Comment by wcyeee** [comment]
去年暑假因為才大一生大二，也沒有找到什麼實習也不打算打工，所以就安心地告訴自己這個暑假好好休息就好！我覺得不要因為看到別人很多活動很認真就開始超級焦慮，如果自己是已經有安排了，那就不要因為看到別人現在在幹嘛就開始焦慮，就照著自己的步調來！然後沒有安排的就可以參考那些很多活動的人，來得及就可以安排進自己的行程，來不及的話，下一個暑假就可以來做！
